Assets/Scripts/StarComponent.cs

```diff
@@ -45,6 +45,7 @@ void OnMouseDown()
         if (!EventSystem.current.IsPointerOverGameObject() && dataController && dataController.starInfoPanel)
         {
             dataController.ChangeConstellationHighlight(starData.Constellation);
+            Debug.Log("(BD=" + starData.BayerDesignation + "),(FD=" + starData.FlamsteedDesignation + "),(ConstShort=" + starData.Constellation + "),(ConstFull=" + starData.ConstellationFullName + ")");
         }

```

Comment:
You can see updated values from the star db by clicking on a star and looking at the console window.